xbmc/video/VideoDatabase.cpp

```diff
@@ -6239,8 +6276,9 @@ bool CVideoDatabase::GetYearsNav(const std::string& strBaseDir, CFileItemList& i
     {
       if (idContent == VIDEODB_CONTENT_MOVIES)
       {
-        strSQL = "select movie_view.premiered, path.strPath, files.playCount from movie_view ";
-        extFilter.AppendJoin("join files on files.idFile = movie_view.idFile join path on files.idPath = path.idPath");
+        strSQL = "SELECT movie_view.premiered, path.strPath, COUNT(history.idFile) FROM movie_view ";
+        extFilter.AppendJoin("JOIN files ON files.idFile = movie_view.idFile JOIN path ON files.idPath = path.idPath");
+        extFilter.AppendJoin("JOIN history ON history.idFile = movie_view.idFile");
```

Comment:
Check if we can use subqueries here instead, as inner join might hide some values